build_tools/ci/cpu_comparison/run_test.sh

```diff
@@ -298,18 +308,66 @@ function run_test() {
   aie_vmfb="${OUTPUT_DIR}/${name}_aie.vmfb"
   cpu_vmfb="${OUTPUT_DIR}/${name}_cpu.vmfb"
 
+
   echo "**** Generating AIE .vmfb file for ${name} ****"
-  ${IREE_COMPILE_EXE} "${test_file}"  \
-      --iree-hal-target-backends=amd-aie \
+  compilation_flags="--iree-hal-target-backends=amd-aie \
       --iree-amdaie-tile-pipeline=${pipeline} \
       --iree-amdaie-matmul-elementwise-fusion \
       --iree-amd-aie-peano-install-dir=${peano_install_path} \
       --iree-amd-aie-mlir-aie-install-dir=${mlir_aie_install_path} \
       --iree-amd-aie-install-dir=${amd_aie_install_path} \
       --iree-amd-aie-vitis-install-dir=${vitis_path} \
       --iree-hal-dump-executable-files-to=$PWD \
-      --mlir-disable-threading \
-      -o "${aie_vmfb}"
+      --mlir-disable-threading -o ${aie_vmfb}"
+
+
+  # TODO(newling) The following logic is copied from run_matmul_test.sh :
+  # factorize out common code in these 2 test scripts to increase maintainability.
+  if [ $use_ukernel -ne 0 ]; then
+
+    compilation_flags="${compilation_flags} --iree-amdaie-enable-ukernels=all"
+
+    #The flag '--iree-amdaie-path-to-ukernels' currently does not work,
+    #see for example https:  // github.com/nod-ai/iree-amd-aie/issues/340
+    #Therefore we need to manually copy(or link) the mm.o file to the
+    #directory in which iree - compile is run.iree - compile is run in the
+    #output directory.Create the softlink only if it is has not already
+    #been created.
```

Comment:
Nit: Please add/remove spaces for this comment.
Question: Will this issue be fixed next?

Reply:
I wasn't planning on looking at it next